Stack/Opc.Ua.Core/Security/Certificates/CertificateTypesProvider.cs

```diff
@@ -134,7 +134,13 @@ public byte[] LoadCertificateChainRaw(X509Certificate2 certificate)
                 return result.Item2;
             }
 
-            return certificate.RawData;
+            // load certificate chain.
```

Comment:
This change contradicts the idea of the cache, which was that certs and certchains are being updated outside the normal create session flows, once. 
Then no async load operation has to be done and no async has to be wired up.

Reply:
Question is why we have a cache miss, it should be loaded once on startup, or when application certificates are updated. handling the cache miss is a band aid.

Reply:
okay, I will rewrite to loop all security policies to update each of the application certificates on certificate update. I did not think the async overhead was such a concern.